src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -20,19 +20,27 @@
     /** The application should switch to event */
     private final boolean event;
 
-    /**PersonListPanel will switch to expanded view showing all details of a friend */
+    /** PersonListPanel will switch to expanded view showing all details of a friend */
     private final boolean showDetails;
 
+    /** The application should switch to insights */
+    private final boolean showInsights;
+
+
 
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, boolean event, boolean showDetails) {
+    public CommandResult(String feedbackToUser, boolean showHelp,
```

Comment:
Maybe we could rename the boolean variables to eg `isShowHelp` / `isExit` in the next iteration! :)